### ISSUE TITLE
Deno validator: set exit code based on validation success/failure

### ISSUE THREAD

**Issue by effigies** [bids-standard/legacy-validator#1909]
Currently always exits 0

**Comment by yarikoptic** [comment]
<details>
<summary>confirming</summary> 

```shell
❯ deno run --allow-read --allow-write --allow-env --allow-net --allow-run /tmp/bids-validator/bids-validator.js 7t_trt --ignoreNiftiHeaders
TypeError: Attempted to load JSON module without specifying "type": "json" attribute in the import statement.
    at async loadSchema (file:///tmp/bids-validator/bids-validator.js:8513:26)
    at async validate (file:///tmp/bids-validator/bids-validator.js:9254:18)
    at async main (file:///tmp/bids-validator/bids-validator.js:9458:24)
    at async file:///tmp/bids-validator/bids-validator.js:9479:1 {
  code: "ERR_MODULE_NOT_FOUND"
}
Warning, could not load schema from https://bids-specification.readthedocs.io/en/latest/schema.json, falling back to internal version
	[WARNING] The 'Authors' field of 'dataset_description.json' should contain an array of values -
with one author per value.
This was triggered based on the presence of only one author field.
Please ignore if all contributors are already properly listed.
 (TOO_FEW_AUTHORS)

		./dataset_description.json
			Evidence: schema.rules.checks.hints.TooFewAuthors

		1 more files with the same issue

	Please visit https://neurostars.org/search?q=TOO_FEW_AUTHORS for existing conversations about this issue.

	[ERROR] Empty files not allowed. (EMPTY_FILE)

		./sub-01/ses-1/anat/sub-01_ses-1_T1map.nii.gz
		./sub-01/ses-1/anat/sub-01_ses-1_T1w.nii.gz
		./sub-01/ses-1/fmap/sub-01_ses-1_run-1_magnitude1.nii.gz

		569 more files with the same issue

	Please visit https://neurostars.org/search?q=EMPTY_FILE for existing conversations about this issue.


          Summary:                          Available Tasks:        Available Modalities:
          731 Files, 75.6 kB                Rest                    MRI                  
          22 - Subjects 2 - Sessions                                                     

	If you have any questions, please post on https://neurostars.org/tags/bids.

changes on filesystem:                                                                                                                                                              
 run_tests.sh | 8 +++++---
❯ echo $?
0

```
</details>

could this issue be prioritized?

**Comment by effigies** [comment]
I have done what I can for now. I think it should be easy, but I would need to spend some time with chatgpt to figure this out.
